public/data/orama254.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Rama Ochieng",
+    "bio": "Designer and Frontend Developer (Ramadevsign) | Co-Organizer #reactdevske | Opensource Maintainer and Contributor",
+    "avatar": "https://github.com/orama254.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/orama254",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/ramaspeaksdev",
+        "icon": "twitter"
+      },
+      {
+        "name": "Check out my Website",
+        "url": "https://ramadevsign.com",
+        "icon": "globe"
+      }
+    ]
+  }
```

Comment:
Extra blank line recommended by GitHub (see red sign)

Reply:
noted. will make a new consolidated commit after a review of the first comment